testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java

```diff
@@ -3996,43 +3931,6 @@ public void verifyFT120() throws IOException, InterruptedException {
                 pregnancyPackJsonPattern, ADMIN_USERNAME, ADMIN_PASSWORD));
     }
 
-    /**
-     * To verify the behavior of Get Inbox Details API if the service is not
-     * deployed in provided Subscriber's state.
-     **/
-    @Test
-    // Similar to FT_16
-    // TODO: https://applab.atlassian.net/browse/NMS-181
-    @Ignore
-    public void verifyFT91() throws IOException, InterruptedException {
```

Comment:
Because it's invalid, I think we both looked at it and realized that you don't check deployment in `getInboxDetail` calls